frontend/src/pages/wmg.tsx

```diff
@@ -46,4 +47,6 @@ const Page = (): JSX.Element => (
   </StylesProvider>
 );
 
+Page.Layout = SidebarLayout;
+
```

Comment:
This was really tricky for me to find/realize, a single short line with a large impact.  Is there anyway we can surface this better? Maybe somehow require layout dependent components to be nested inside a component where `Layout` is defined? Similar to a Context Provider

Reply:
Hmm thanks for raising this! Yeah, right now you'd need to look at how other pages use custom Layout and check out `_app.tsx` to see how `Page.Layout` is used 🤔 

I think maybe let's start with adding a comment above `Page.Layout = SidebarLayout;` to explain what it does, so it  brings attention to the reader? And then when we have time in cool time period, we can figure out a structural way to solve this

The reasoning is because we don't add pages frequently, so hopefully comment is enough as a patch for now

What do you all think?

Thank you!

Reply:
You're right.  If we aren't creating pages often, if any this Q, it's not worth putting time into this just yet.

Reply:
```suggestion
// This layout needs to be defined in order to correctly style the SideBars and their wrappers
Page.Layout = SidebarLayout;

```